angular-workspace/projects/ni/nimble-angular/src/directives/radio-group/nimble-radio-group.directive.ts

```diff
@@ -0,0 +1,57 @@
+import { Directive, ElementRef, Input, Renderer2 } from '@angular/core';
+import { Orientation, RadioGroup } from '@ni/nimble-components/dist/esm/radio-group';
+import { BooleanValueOrAttribute, toBooleanProperty } from '../utilities/template-value-helpers';
+
+export { Orientation };
+export type { RadioGroup };
+
+/**
+ * Directive to provide Angular integration for the radio group.
+ */
+@Directive({
+    selector: 'nimble-radio-group'
+})
+export class NimbleRadioGroupDirective {
+    private _nextOpenId = 0;
+
+    public get disabled(): boolean {
+        return this.elementRef.nativeElement.disabled;
+    }
+
+    @Input() public set disabled(value: BooleanValueOrAttribute) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'disabled', toBooleanProperty(value));
+    }
+
+    public get name(): string {
+        return this.elementRef.nativeElement.name;
+    }
+
+    @Input() public set name(value: string) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'name', value);
+    }
+
+    public get orientation(): Orientation {
+        return this.elementRef.nativeElement.orientation;
+    }
+
+    @Input() public set orientation(value: Orientation) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'orientation', value);
+    }
+
+    public get value(): string {
+        return this.elementRef.nativeElement.value;
+    }
+
+    @Input() public set value(value: string) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'value', value);
+    }
```

Comment:
We should not expose anything about the radio-group value to users. Users should only be manipulating radio values via the CVA. We should remove these.
Reasons:
- If the user assigns a value here it may be out of sync with the CVA
- It will not be a useful value as currently implemented, just the generated allocateIDs

Reply:
I removed `value` from the radio group directive. It was only used by the CVA, but I've changed that so that it sets the value directly on the radio group element, not via the directive.